org.jacoco.doc/docroot/index.html

```diff
@@ -68,6 +68,12 @@
       <td>Ant <i>(all other dependencies included)</i></td>
     </tr>
     <tr>
+      <td><span class="el_jar">jacococli.jar</span></td>
+      <td>no</td>
+      <td>JaCoCo Command Line Interface</td>
```

Comment:
@Godin Thanks for bringing this point up! What classifers do/should we actually deploy here? The plain jar or the nodeps variant?

Also what is missing here is the `nodeps` classifier on `org.jacoco.ant` and the `runtime` classifier on `org.jacoco.agent`. It looks like the table requires a new "classifer" column. Cleanup in a separate PR?